integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoQueryIT.java

```diff
@@ -61,80 +66,97 @@
 public class TrinoQueryIT {
   private static final Logger LOG = LoggerFactory.getLogger(TrinoQueryIT.class);
 
-  private static boolean isDockerRunning = false;
+  private static boolean autoStartEnv = true;
 
   // TODO(yuhui) redo get the configs after we have the Docker image ready for testing.
   private static String gravitinoUri = "http://127.0.0.1:8090";
   private static String trinoUri = "http://127.0.0.1:8080";
   private static String hiveMetastoreUri = "thrift://127.0.0.1:9083";
-  private static String mysqlUri = "jdbc:mysql://127.0.0.1?useSSL=false";
-  private static String postgresqlUri = "jdbc:postgresql://127.0.0.1/mydb";
+  private static String hdfsUri = "hdfs://127.0.0.1:9000";
+  private static String mysqlUri = "jdbc:mysql://127.0.0.1";
+  private static String postgresqlUri = "jdbc:postgresql://127.0.0.1";
+
+  private static GravitinoClient gravitinoClient;
+  private static TrinoITContainers trinoITContainers;
+  private static TrinoQueryRunner trinoQueryRunner;
 
   private static final String metalakeName = "test";
+  private static GravitinoMetaLake metalake;
   private static final Set<String> testCatalogs = new HashSet<>();
 
-  private static GravitinoClient client;
-  private static GravitinoMetaLake metalake;
   private static String testQueriesDir = "";
   private static AtomicInteger testCount = new AtomicInteger(0);
   private static AtomicInteger totalCount = new AtomicInteger(0);
-  static TrinoQueryRunner trinoQueryRunner;
+
+  private static void setEnv() throws Exception {
+    if (autoStartEnv) {
+      AbstractIT.startIntegrationTest();
+      trinoITContainers = ContainerSuite.getInstance().getTrinoITContainers();
+      trinoITContainers.launch(AbstractIT.getGravitinoServerPort());
+      gravitinoClient = AbstractIT.getGravitinoClient();
+
+      gravitinoUri =
+          String.format("http://%s:%d", "127.0.0.1", AbstractIT.getGravitinoServerPort());
```

Comment:
I think better change to `String.format("http://127.0.0.1:%d", AbstractIT.getGravitinoServerPort());`?